nomad/state/state_store_test.go

```diff
@@ -2958,6 +2969,11 @@ func TestStateStore_CSIVolume(t *testing.T) {
 	require.True(t, vs[0].ReadSchedulable())
 
 	// Deregister
```

Comment:
This comment line seems to be a stray after this change.

Reply:
Yep, it's gone!